althea_types/src/interop.rs

```diff
@@ -736,3 +736,9 @@ pub struct HeartbeatMessage {
     /// The router version stored in semver format as found in the Cargo.toml
     pub version: String,
 }
+
+/// An exit's unix time stamp that can be queried by a downstream router
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ExitTimeStamp {
+    pub secs_unix_epoch: i64,
```

Comment:
Why not use a SystemTime Struct?  https://doc.rust-lang.org/std/time/struct.SystemTime.html

It's mostly just a u64 with associated functions.

Reply:
will do